src/tfr_msgs/action/Navigation.action

```diff
@@ -0,0 +1,22 @@
+#goal msg
+Header header
+uint8 location_code
+uint8 UNSET = 0
+uint8 TO_MINING = 1
+uint8 TO_DUMPING = 2
+---
+#feedback msg
+Header header
+uint8 status
+uint8 OK = 0
+uint8 ERROR = 1
+geometry_msgs/Pose current
+geometry_msgs/Pose goal
+---
+#result msg 
+Header header
+uint8 status
+uint8 OK = 0
+uint8 FAILURE = 1
+geometry_msgs/Pose current
+geometry_msgs/Pose goal
```

Comment:
I don't understand the format here:
```
uint8 <status_code>
uint8 CODE1 = 0
uint8 CODE2 = 1
uint8 CODE3 = 2
```
It seems like we're sending a byte for status code, which sounds fine, but then we're also sending the values for what that status code means?  I assume the `=` operator is for default values.  Is there a reason to send this extra information?

Reply:
Thats how you declare a message level constant those aren't acutally sent over the net. It gets saved in an enum like device in the namespace of the generated message header.

Reply:
Ah.  That's fine, then.